pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.pylint.master]
+
+exit-zero = true
+errors-only = true
```

Comment:
Drop this section, this will be included as CLI argument in the workflow, and it is the content of the two separate calls to Pylint:
1. the first one is `errors-only`, and it should make the CI crash in case there is an actual error
2. the second one includes also the warnings, but it has to be `exit-zero`, since you do not want to make tests failing for a Pylint warning

Reply:
```suggestion
```